core/src/banking_stage/transaction_scheduler/scheduler_common.rs

```diff
@@ -27,17 +27,31 @@ pub struct Batches<Tx> {
 
 impl<Tx> Batches<Tx> {
     pub fn new(num_threads: usize, target_num_transactions_per_batch: usize) -> Self {
```

Comment:
since we now store `Batches` in common, we don't need to do the `0` target size to avoid allocation on the final send. So the `target_num_transactions_per_batch` is constant.

We could store it in the `Batches` itself now, making all the calls to `send_*` a bit cleaner.

Reply:
done!